package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Utilities to help when developing terra modules.",
   "main": "lib/index.js",
   "engines": {
-    "node": ">=8.9.2"
+    "node": ">=8.9.2 <12"
```

Comment:
Should we make this `>= 8.10.0 <12` due to this issue https://github.com/cerner/eslint-config-terra/issues/42

Reply:
..... maybe??

Reply:
https://github.com/cerner/terra-toolkit/pull/391/commits/1c1d4bee1491e0e6f18a5cb8fed7a4b3d6dfdf53